3scale/porta
# THREESCALE-9327 Quick Start content- Main Page Part 1

**What this PR does / why we need it**:

We have too many Quick starts and we have to split them into different containers. This is part 1 of the improvements designed at [THREESCALE-9327](https://issues.redhat.com/browse/THREESCALE-9327)

**Which issue(s) this PR fixes** 

[THREESCALE-9327](https://issues.redhat.com/browse/THREESCALE-9327)

**Verification steps** 

Go to Quick Starts and confirm there are three different containers (Learn about 3scale features, Common instances and Learning resources).

**Before**
![Captura de pantalla 2023-11-17 a las 12 57 44](https://github.com/3scale/porta/assets/72191439/e239e75d-ca4e-4c3f-a447-fc34b2d6a185)

**After**
<img width="1785" alt="Captura de pantalla 2023-12-11 a las 11 17 36" src="https://github.com/3scale/porta/assets/72191439/58e80e72-28f0-444e-8ad2-3408fe401f13">
